mod-kb-ebsco/mod-kb-ebsco-java.postman_collection.json

```diff
@@ -2402,6 +2402,1277 @@
+													"            pm.expect(firstRecord.attributes.name).to.have.string('Search');",
+													"        })",
+													"    } else {",
+													"        console.log(\"No packages returned for this provider with this search query\");",
+													"    }",
+													"}",
+													""
+												],
+												"type": "text/javascript"
+											}
+										}
+									],
+									"request": {
+										"method": "GET",
+										"header": [
+											{
+												"key": "x-okapi-tenant",
+												"value": "{{xokapitenant}}"
+											},
+											{
+												"key": "x-okapi-token",
+												"value": "{{xokapitoken}}"
+											}
+										],
+										"body": {
+											"mode": "raw",
+											"raw": ""
+										},
+										"url": {
+											"raw": "{{protocol}}://{{url}}:{{okapiport}}/eholdings/providers/19/packages?q=Search",
+											"protocol": "{{protocol}}",
+											"host": [
+												"{{url}}"
+											],
+											"port": "{{okapiport}}",
+											"path": [
+												"eholdings",
+												"providers",
+												"19",
+												"packages"
+											],
+											"query": [
+												{
+													"key": "q",
+													"value": "Search"
+												}
+											]
+										}
+									},
+									"response": []
+								},
+								{
+									"name": "with q and count",
+									"event": [
+										{
+											"listen": "test",
+											"script": {
+												"id": "48432b4d-f7c2-4f08-b9e9-8f1eac8b921c",
+												"exec": [
+													"pm.test(\"success test\", function() {",
+													"    pm.response.to.be.json;",
+													"});",
+													"",
+													"let response = pm.response.json();",
+													"",
+													"//Check that status is 200",
+													"pm.test(\"Status is 200\", function () {",
+													"    pm.response.to.have.status(200);",
+													"});",
+													"",
+													"//Validate response against json api schema",
+													"pm.test(\"Validate schema\", function () {",
+													"     tv4.addSchema(\"JSONAPI.json\", JSON.parse(pm.variables.get(\"schema_jsonapi_content\")));",
+													"    pm.expect(tv4.validate(response, JSON.parse(pm.environment.get(\"schema_jsonapi_content\")))).to.equal(true, \"Schema validation error: \" + JSON.stringify(tv4.error));",
+													"    pm.expect(tv4.missing.length).to.equal(0, \"Missing schemas: \" + JSON.stringify(tv4.missing));",
+													"});",
+													"",
+													"//Check if we get a collection of providers in response",
+													"if(response.data) {",
+													"    let len = response.data.length;",
+													"    if(len > 0){",
+													"        //Test that number of records returned are less than or equal to count",
+													"        //This should be re-visited after https://issues.folio.org/browse/UIEH-466 is fixed.",
```

Comment:
It seems that this comment is also outdated, because java module passes correct count to RM API

Reply:
Thanks, updated